plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java

```diff
@@ -44,18 +44,6 @@
 
     private DeltaLakeQueryRunner() {}
 
-    public static DistributedQueryRunner createDeltaLakeQueryRunner()
```

Comment:
let's keep this one, since it seems common

Reply:
Updated.